azurerm/internal/services/policy/policy_assignment_resource.go

```diff
@@ -339,3 +347,11 @@ func expandAzureRmPolicyNotScopes(d *schema.ResourceData) *[]string {
 
 	return &notScopesRes
 }
+
+func convertEnforcementMode(mode *bool) policy.EnforcementMode {
```

Comment:
We'll need to update this too if we remove the pointer up above.

Reply:
that was clear, I did that